robotics-ai-suite/robot-vision-control/docs/source/substitutions.txt

```diff
@@ -17,19 +17,23 @@
 
 .. |ros| replace:: ROS 2
 .. |ros_version| replace:: Humble
+.. |ROS2-jazzy| replace:: ROS 2 Jazzy Jalisco
```

Comment:
This file is not used during the build of the documentation for OEP.  All substitutions in the documentation should be replaced.